### DISCUSSION TITLE
Version at PyPI still has a Django <= 3.1 requirement

### DISCUSSION THREAD

**Discussion by MrCordeiro** [wagtail/wagtail-localize#443] [General]
Hello,
It would be nice if the code on PyPI could be updated to accept Django >= 3.2. Would it break anything?

**Comment by kaedroho** [comment]
Hey @MrCordeiro 

I'm not sure, but I don't have much time to maintain this project right now. Would you (or anyone else reading) be up for trying this out and submitting a PR?

Version needs to be updated in:

 - setup.py (requirements and classifiers)
 - tox.ini
 - .github/workflows/test.yml
 - readme.md

**Reply by MrCordeiro** [reply]
Hey @kaedroho! I think that the Github code is ok. I can see the files you mentioned already support the newest Django version. It's just the PyPI code that lacks Django 3.2

**Reply by kaedroho** [reply]
Ahh, sorry I thought you meant Django 4.0! Wagtail localize 1.0RC3 supports Django 3.2 and this is on PyPI

**Comment by 2567910** [comment]
For me it is still not working on Django 3.2.

**Reply by MrCordeiro** [reply]
Are you using the release candidate version, like @kaedroho mentioned?

`pip install wagtail-localize==1.0rc4`
